processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -30,17 +30,37 @@
 import org.joda.time.Interval;
 
 import java.util.Arrays;
+import java.util.EnumSet;
 import java.util.Map;
 
 public class SegmentMetadataQuery extends BaseQuery<SegmentAnalysis>
 {
+  public enum AnalysisType
+  {
+    CARDINALITY,
+    SIZE;
+
+    @JsonCreator
+    public static AnalysisType fromString(String name) {
+      return valueOf(name.toUpperCase());
+    }
```

Comment:
@xvrl Added an annotated @JsonValue toString method for AnalysisType, updated serde test for analysisTypes field